src/foremast/awslambda/awslambda.py

```diff
@@ -91,6 +107,39 @@ def _get_sg_ids(self):
             sg_ids.append(sg_id)
         return sg_ids
 
+    @retries(max_attempts=3, wait=1)
```

Comment:
Include an `exceptions=(boto3.exceptions.botocore.exceptions.ClientError)` in `retries` so we don't retry accidentally with `KeyError` or something else.